pkg/image/apis/image/types.go

```diff
@@ -439,6 +439,12 @@ type ImageStreamTag struct {
 
 	// The Image associated with the ImageStream and tag.
 	Image Image
+
+	// PublicImageReference is the pull spec that can be used to acccess this
+	// image from outside of the cluster. It may be empty if the cluster is not
+	// configured to expose an external image reference.
+	// +optional
+	PublicImageReference string
```

Comment:
where does this value actually come from?  ImageStreamTag is a virtual resource, right?  (no one actually creates an ImageStreamTag, they create TagReferences and TagEvents within imagestreams)